Plugins/Public/playercntl_plugin/Rename.cpp

```diff
@@ -455,6 +458,15 @@ namespace Rename
 
 				// The rename worked. Log it and save the rename time.
 				AddLog("NOTICE: User rename %s to %s (%s)", wstos(o.wscCharname).c_str(), wstos(o.wscNewCharname).c_str(), wstos(HkGetAccountID(acc)).c_str());
+
+				wstring wscCharFileName2;
+				HkGetCharFileName(o.wscNewCharname, wscCharFileName2);
+				//Anti renameme, for /resetrep cooldown
+				if (MiscCmds::ShipnamesWhichAlreadyUsedResetRep.find(wscCharFileName) != MiscCmds::ShipnamesWhichAlreadyUsedResetRep.end())
```

Comment:
So we're actually using char file names in this map instead of char names?

Reply:
yup, more reliable. char file names are having always names like abdfbd_dfdsf
only English alphabet with one "_"
I feel it is more secure to use than all the random things a regular player name can be.

Reply:
okay so we should name it that way

Reply:
it was already named as CharFileName. Not sure what you mean here

Reply:
Right now this is named 'shipnames' (i.e., character names), but actually it's char file names. Not the same thing.

Reply:
Uh. I guess I was blind to it. Fixed.
https://github.com/DiscoveryGC/FLHook/pull/138/commits/fcc87ffc1733218a9972786c512407c5ca6b9ed9